client/extensions/woocommerce/components/list/list-table/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+
+class ListTable extends Component {
+
+	render() {
+		const { children } = this.props;
+		return (
+			<Card className="list-table">
+				<table className="list-table__table">
```

Comment:
Using `<table>` gives me the chills. I know this is tabular data, but still... Let's see what the designers say of this, I don't know if `<table>`s are still considered super-evil.